proxy/main.ts

```diff
@@ -37,7 +46,16 @@ app.addEventListener("listen", (event) => {
 });
 
 app.addEventListener("error", (evt) => {
-  console.log("error", evt.error);
+  const ctx = evt.context;
+  if (ctx) {
+    const status = ctx.response.status;
+    console.log(
```

Comment:
nit: maybe use `console.error` for errors?

Reply:
It doesn't render any differently - I personally think `console.error` is pretty useless :upside_down_face:

Reply:
I agree with the rendering part - I just thought it might be better to send errors to stderr for easy access.

Reply:
Ah I see - didn't know we print errors to stderr :upside_down_face:.